cvat/schema.yml

```diff
@@ -8880,17 +8953,11 @@ components:
       - error_count
       - frame_count
       - frame_share
+      - frames_with_errors
       - gt_count
+      - total_frames
       - valid_count
       - warning_count
-    QualityReportTarget:
```

Comment:
It seems drf-spectacular will map all the enums with the same members into 1 type. I think since we just launched the feature, it's safe enough to change API type names.

Reply:
This seems like dangerous behavior. Suppose we add another possible target for analytics, but not quality (or vice versa) - will the enum split back into two again? Is there no way to ask drf-spectacular to not merge enums?